index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
+collection of data. There are at least two types of data schemas that this
+specification considers:
+        </p>
+
+        <ul>
+          <li>
+Data verification schemas, which are used to <a>verify</a> that the structure
+and contents of a <a>credential</a> or <a>verifiable credential</a> conform to a
+published schema.
+          </li>
+          <li>
+Data encoding schemas, which are used to map the contents of a
+<a>verifiable credential</a> to an alternative representation format, such as a
+format used in a zero-knowledge proof.
+          </li>
+        </ul>
+
+        <p>
+It is important to understand that data schemas serve a different purpose from
+the <code>@context</code> property, which neither enforces data structure or
+data syntax, nor enables the definition of arbitrary encodings to alternate
+representation formats.
+        </p>
+        <p>
+This specification defines the following <a>property</a> for the expression of a
+data schema, which can be included by an <a>issuer</a> in
+the <a>verifiable credentials</a> that it issues:
+        </p>
+
+        <dl>
+          <dt><var>credentialSchema</var></dt>
+          <dd>
+            <p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
+more data schemas that provide <a>verifiers</a> with enough information to
+determine if the provided data conforms to the provided schema(s). Each
+<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
+<code>JsonSchema2023</code>), and an <code>id</code> <a>property</a>
+that MUST be a <a>URL</a> identifying the schema file. The precise contents of
+each data schema is determined by the specific type definition.
+            </p>
+            <p>
+If multiple schemas are present, validity is determined according to the
+processing rules outlined by each associated <code>credentialSchema</code>
+<code>type</code> property.
+            </p>
+          </dd>
+        </dl>
+
+        <p class="note">
+The <code>credentialSchema</code> <a>property</a> provides an opportunity to
+annotate type definitions or lock them to specific versions of the vocabulary.
+Authors of <a>verifiable credentials</a> can include a static version of their
+vocabulary using <code>credentialSchema</code> that is locked to some content
+integrity protection mechanism. The <code>credentialSchema</code>
+<a>property</a> also makes it possible to perform syntactic checking on the
+<a>credential</a> and to use <a>verification</a> mechanisms such as JSON Schema
+[[?VC-JSON-SCHEMA-2023]] validation.
+        </p>
+
+        <pre class="example nohighlight"
+          title="Usage of the credentialSchema property to perform JSON schema validation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential", "ExamplePersonCredential"],
```

Comment:
```suggestion
  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
```

Reply:
or are you thinking that every credential schemas asserts the presence of a specific array element... without knowing its index?